tap_ilevel/transform.py

```diff
@@ -1,42 +1,19 @@
-import re
-
-# Convert camelCase to snake_case
-def convert(name):
-    regsub = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
-    return re.sub('([a-z0-9])([A-Z])', r'\1_\2', regsub).lower()
-
-
-# Convert keys in json array
-def convert_array(arr):
-    new_arr = []
-    for i in arr:
-        if isinstance(i, list):
-            new_arr.append(convert_array(i))
-        elif isinstance(i, dict):
-            new_arr.append(convert_json(i))
-        else:
-            new_arr.append(i)
-    return new_arr
+#!/usr/bin/env python3
```

Comment:
This seems strange - is it necessary?

Reply:
Ok - removed. When is this header needed?